app/views/puppetclasses/_class_parameters.html.erb

```diff
@@ -16,6 +16,9 @@
     class="fields <%= 'overridden' if overridden %>">
     <%= content_tag(:td, puppetclass.name, :rowspan => lookup_keys.size, :class => "ellipsis") if index == 0 %>
     <td class="ellipsis param_name">
```

Comment:
`param_name` class should be on the other column that contains the param name (lookup_key.key). Both columns should have the `ellipsis` class so they get a tooltip if they overflow (e.g. on a small display). I think the order should be the other way around, first name then type.